omod/src/test/java/org/openmrs/module/owa/web/controller/OwaRestControllerTest.java

```diff
@@ -138,4 +138,10 @@ public void install_wrongDownloadUrl() throws Exception {
 		List<App> appList = controller.install(requestData, request, response);
 		Assert.assertEquals("owa.not_a_zip", request.getSession().getAttribute(WebConstants.OPENMRS_ERROR_ATTR));
 	}
+
+	@Test
+	public void allowWebAdmin_shouldNotReturnNull() {
+		OwaRestController controller = (OwaRestController) applicationContext.getBean("owaRestController");
```

Comment:
The test should not just just for not null. And it should also be a test for making a real or at least mock rest call.

Reply:
i will improve the tests

Reply:
Can't you return without first assigning to a variable? That is what i mean by saying that is variable is unused (useless)